Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -113,7 +116,7 @@ def http_request(
         Returns:
             Response from api according to resp_type. The default is `json` (dict or list).
         """
-        if 'ok_codes' not in kwargs:
+        if 'ok_codes' not in kwargs and not self._ok_codes:
```

Comment:
Maybe im missing something here, but where are we initializing the `_ok_codes` attribute? that should be `ok_codes` (in BaseClient as well)

Reply:
i initialized it in the AzureSentinel constructor
it should use the _ok_codes from the client instance instead of sending it as arg in each http_request
the ok_codes argument is for override the default ok_codes in specific request